substrate/frame/tx-pause/src/lib.rs

```diff
@@ -15,6 +15,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! # Transaction Pause
+//!
+//! Allows dynamic chain state based pausing and unpausing of specific extrinsics via call filters.
+//!
+//! ## Pallet API
+//!
+//! See the [`pallet`] module for more information about the interfaces this pallet exposes,
+//! including its configuration trait, dispatchables, storage items, events and errors.
+//!
+//! ## Overview
+//!
+//! Think of it as a dynamic call filter that can be controlled with extrinsics.
```

Comment:
Too conversational for reference docs.

Reply:
Yeah. I had lifted it from the PR. Updated to read more clearly.